main.py

```diff
@@ -1,10 +1,47 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as data_file:
+        data_source = json.load(data_file)
+
+    for player, info in data_source.items():
+
+        race_info = info["race"]
+        skill_info = info["race"]["skills"]
+        guild_info = info["guild"]
+
+        race, created = Race.objects.get_or_create(
```

Comment:
You do not use the `created` variable defined here and on line 31, so you could omit the value by using `_`.